recipes/allo/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set version = "1.0.4" %} 
+
+package:
+  name: allo
+  version: {{ version }}
+  
+source:
+  url: https://files.pythonhosted.org/packages/5f/17/7738e47da69e5227c9141c78d5f6806806c8d88c473d6782ffb75bb70c04/bio-allo-{{ version }}.tar.gz
+  sha256: d18b54ac566fef33c5b491ff1eec49dc98d25c218b1b1a263c5384743b580c2f
+
+build:
+  run_exports:
+    - {{ pin_subpackage('allo', max_pin="x") }}
+  number: 0
+  noarch: generic
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python ==3.10
+  run:
+    - python ==3.10
+    - numpy ==1.23
+    - joblib ==1.1.0
+    - tensorflow ==2.11
+    - pysam ==0.20.0
```

```suggestion
    - python
    - numpy
    - joblib
    - tensorflow
    - pysam
```

Please do not use strict-pins. If pins are necessary, please use lower and upper bounds.